app/templates/main/map.html

```diff
@@ -57,14 +57,16 @@
                                 '<button id="close_info" class="ui icon button close">' +
                                 '<i class="remove icon"></i>' +
                                 '</button>' +
-                                '<h3>Date: ' + '{{ report.date }}' + '</h3>' +
-                                '<p>Duration: ' + '{{ report.duration }}' + '</p>' +
+                                '<h3>Date: ' + '{{ report.date.strftime('%m-%d-%Y at %I:%M %p') }}' + '</h3>' +
+                                '<p>Duration (h:m:s): ' + '{{ report.duration }}' + '</p>' +
                                 '<p>Vehicle ID: ' + '{{ report.vehicle_id }}' +'</p>' +
-                                '<p>License Plate: ' + '{{ report.license_plate }}' + '</p>' +
-                                {% if current_user.is_admin() or report.agency in current_user.agencies or report.agency.is_public %}
+                                '<p>License Plate: ' + '{{ report.license_plate if report.license_plate else '' }}' + '</p>' +
+                                {% if current_user.is_admin() or report.agency in current_user.agencies or report.show_agency_publicly %}
```

Comment:
> Or is the report being shown but the agency name being hidden/kept private? In which case, would suggest report.has_public_agency or report.agency_is_public

Yes it's this. I'll take a look at the documentation but I personally think the variable name is fine.